src/Aspire.Dashboard/Resources/Dialogs.resx

```diff
@@ -0,0 +1,176 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="FilterDialogFieldPlaceholder" xml:space="preserve">
+    <value>Field</value>
+  </data>
+  <data name="FilterDialogTextValuePlaceholder" xml:space="preserve">
+    <value>Value</value>
+  </data>
+  <data name="FilterDialogCancelButtonText" xml:space="preserve">
+    <value>Cancel</value>
+  </data>
+  <data name="FilterDialogApplyFilterButtonText" xml:space="preserve">
+    <value>Apply Filter</value>
+  </data>
+  <data name="FilterDialogRemoveFilterButtonText" xml:space="preserve">
+    <value>Remove Filter</value>
+  </data>
+  <data name="FilterConditionContains" xml:space="preserve">
+    <value>contains</value>
+  </data>
+  <data name="FilterConditionNotContains" xml:space="preserve">
+    <value>not contains</value>
+  </data>
+  <data name="FilterFieldMessage" xml:space="preserve">
+    <value>Message</value>
+    <comment>Message is a field name, not an action</comment>
+  </data>
+  <data name="FilterFieldApplication" xml:space="preserve">
+    <value>Application</value>
+  </data>
+  <data name="FilterFieldTraceId" xml:space="preserve">
+    <value>TraceId</value>
+  </data>
+  <data name="FilterFieldSpanId" xml:space="preserve">
+    <value>SpanId</value>
+  </data>
+  <data name="FilterFieldOriginalFormat" xml:space="preserve">
+    <value>OriginalFormat</value>
+  </data>
+  <data name="FilterFieldCategory" xml:space="preserve">
+    <value>Category</value>
+  </data>
```

Comment:
As mentioned above, I suspect we shouldn't localise field names. Please confirm!